docker-compose.yml

```diff
@@ -49,12 +62,12 @@ services:
       BLACKFIRE_ENABLE: ${BLACKFIRE_ENABLE:-0}
       BLACKFIRE_SERVER_ID: ${BLACKFIRE_SERVER_ID:-0}
       BLACKFIRE_SERVER_TOKEN: ${BLACKFIRE_SERVER_TOKEN:-0}
-    command: ["/tmp/wait-for-it.sh", "--timeout=60", "--strict", "mysql:3306", "--", "/tmp/docker_run_git.sh"]
+    command: /tmp/wait-for-it.sh --timeout=60 --strict ${DB_SERVER:-mysql}:3306 -- /tmp/docker_run_git.sh
     ports:
       - "8001:80"
       - "8002:443"
     depends_on:
-      - mysql
+      - "${DB_SERVICE:-mysql}"
```

Comment:
Why the need for two variables? `DB_SERVER` and `DB_SERVICE` which seem to always have the same value
Couldn't it be just one variable?

Reply:
Correct. Fixed